libstatistics_collector/include/libstatistics_collector/topic_statistics_collector/constants.hpp

```diff
@@ -17,6 +17,8 @@
 
 #include <string>
 
+#include "libstatistics_collector/visibility_control.hpp"
```

Comment:
Needed? Visibility macros don't seem to be used in this file?

Same comment for all files that include the header but don't use the macros.

Reply:
Will remove where unused.